docs/migrations/chain-upgrade-guide-040.md

```diff
@@ -60,12 +64,12 @@ software and restore to their latest snapshot before restarting their nodes.
 1. Migrate the exported state to `github.com/cosmos/cosmos-sdk@0.40.*` compatible genesis state.
 
    ```shell
-   simd migrate v0.40 v039_exported_state.json --chain-id <new_chain_id> --genesis-time <new_genesis_time_in_utc> > new_v040_genesis.json
+   simd migrate v0.42 v039_exported_state.json --chain-id <new_chain_id> --genesis-time <new_genesis_time_in_utc> > new_v042_genesis.json
```

Comment:
This was changed in https://github.com/cosmos/cosmos-sdk/pull/8880#discussion_r594423667